bigflow/cli.py

```diff
@@ -543,63 +545,39 @@ def _cli_deploy_image(args):
 def find_image_file():
     # TODO parametrize ".image" using settings from build.py
     files = glob1(".image", "*-*.tar")
+    print("Z" * 100, list(files))
```

Comment:
What is purpose of this print?

Reply:
This line prints very nice and beautiful comment. :D
```
ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
```

Ok, it was just a debug print :)